src/backend/TrafficCourts/Citizen.Service/Program.cs

```diff
@@ -46,3 +53,24 @@
 {
     Serilog.Log.CloseAndFlush();
 }
+
+
+void AddMockInvoiceEndpoint(WebApplication application)
+{
+    var type = builder.Configuration.GetSection($"TicketSearch:SearchType").Get<TicketSearchType>();
+    if (type == TicketSearchType.Mock)
+    {
+        application.MapPost("/api/invoice", [AllowAnonymous] (Invoice invoice, [FromServices] IMemoryCache cache) =>
```

Comment:
Will Max need a DELETE endpoint as well to clean up the memory cache after his unit test? I suppose we can just restart the api to reset the cache.

Reply:
I had suggested saving the entries with expiration so they would be cleared out after a period of time, say a few hours or a day. This way, there is no need to delete. However, lets get this in and see if there are any adjustments required.